explore/src/main/scala/explore/TopBar.scala

```diff
@@ -56,7 +70,34 @@ object TopBar {
               MenuItem(
                 <.span(
                   ExploreStyles.MainTitle,
-                  "Explore"
+                  "Explore",
+                  Popup(
+                    on = Click,
+                    basic = true,
+                    position = RightCenter,
+                    size = Small,
+                    clazz = ExploreStyles.DiscretePopup,
+                    trigger = IconGroup(clazz = ExploreStyles.Info)(
+                      Icons.Info.copy(corner = TopRight)
+                    )
+                  )(
+                    PopupContent(
+                      <.span(ExploreStyles.Version)(
+                        s"Version: ${appCtx.version}",
+                        CopyToClipboard(
+                          text = appCtx.version.value,
+                          onCopy = (_, copied) =>
+                            $.setState(State(copied)) >>
+                              $.setState(State(false)).delayMs(1500).toCallback
```

Comment:
The temporary check mark is a nice touch.

Reply:
Yeah, much better than the toast we have on the seqexec